rest/pom.xml

```diff
@@ -202,13 +202,12 @@
     </plugins>
   </build>
 
-  <profiles>    
+  <profiles>
      <profile>
         <id>auth</id>
         <activation>
             <property>
                 <name>auth</name>
```

Comment:
Then leave as is (-D) for 0.8 and try to make it work with -P for the next release.